test/evergreen.yml

```diff
@@ -3008,6 +3036,105 @@ tasks:
+            set -o verbose
+
+            # Run AZURE extension unit testing.
+            ext/storage_sources/azure_store/test/run_azure_unit_tests
+  - name: gcp-tiered-unittest-test
+    tags: ["pull_request", "tiered_unittest"]
+    commands:
+      - func: "get project"
+      - func: "install gcp dependencies"
+      - func: "compile wiredtiger"
+        vars:
+          <<: *configure_flags_tiered_storage_gcp
+          HAVE_UNITTEST: -DHAVE_UNITTEST=1
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          include_expansions_in_env:
+            - gcp_sdk_ext_private_key
+            - gcp_sdk_ext_private_key_id
+          script: | 
+            set -o errexit
+            # GCP requires a path to a credentials file for authorization. To not expose the private key within the file, 
+            # use a placeholder private key. The placeholders are then replaced in the command line with the evergreen expansion variables 
+            # and stored in a temporary file. The file is deleted after the unit tests are run.
+            file=$(mktemp --suffix ".json")
+            sed 's|gcp_private_key|'"${gcp_sdk_ext_private_key}"'|;s|gcp_private_id|${gcp_sdk_ext_private_key_id}|' ../test/evergreen/gcp_auth.json >> $file            
+            export GOOGLE_APPLICATION_CREDENTIALS="$file"
+            set -o verbose
+
+            # Run GCP extension unit testing.
+            ext/storage_sources/gcp_store/test/run_gcp_unit_tests
+  - name: azure-tiered-test-small
+    tags: ["pull_request", "python"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          <<: *configure_flags_tiered_storage_azure
+          cmake_binary: /opt/mongodbtoolchain/v4/bin/cmake
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          include_expansions_in_env:
+            - azure_sdk_ext_access_key
+          script: |
+            set -o errexit
+            # Set the Azure credentials using config variable.
+            export AZURE_STORAGE_CONNECTION_STRING="${azure_sdk_ext_access_key}"
+            set -o verbose
+
+            ${test_env_vars|} python3 ../test/suite/run.py -j $(nproc) tiered19
+  - name: gcp-tiered-test-small
+    tags: ["pull_request", "python"]
+    commands:
+      - func: "get project"
+      - func: "install gcp dependencies"
+      - func: "compile wiredtiger"
+        vars:
+          <<: *configure_flags_tiered_storage_gcp
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          include_expansions_in_env:
+            - gcp_sdk_ext_private_key
+            - gcp_sdk_ext_private_key_id
+          script: |
+            set -o errexit
+            # GCP requires a path to a credentials file for authorization. To not expose the private key within the file, 
```

Comment:
Can you format this comment a little more nicely please? The lines are (a) quite long, and (b) inconsistent lengths.

Reply:
Looks like it is strictly the same comment as the one on line 3080. Maybe we could avoid duplication and find the right spot for this comment.

Reply:
Done! [5f6589f](https://github.com/wiredtiger/wiredtiger/pull/8842/commits/5f6589fa932fdb03b4dc0351fae386913ae89d4e)

Reply:
Hey @jiechenbo, this comment has not been changed while it was changed [here](https://github.com/wiredtiger/wiredtiger/pull/8842/files#diff-42f20e21217bd7d4b14489056e1ea24ea2b87599c85413d49182a3b0b721c132R3075).

Reply:
Thanks missed it, fixed it here [f4ccca9](https://github.com/wiredtiger/wiredtiger/pull/8842/commits/f4ccca9b20ac942fef206a19243b8c6af6266259)